src/test/java/com/spotify/github/v3/clients/RepositoryClientTest.java

```diff
@@ -79,8 +89,10 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.ArgumentCaptor;
+import org.mockito.ArgumentMatchers;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
+import uk.co.datumedge.hamcrest.json.SameJSONAs;
```

Comment:
Saw it on classpath, okay to use? Do not want to worry about difference in whitespace/order in mock and real-deal.